src/test/java/org/opentripplanner/routing/algorithm/mapping/__snapshots__/TransitSnapshotTest.snap

```diff
@@ -4032,7 +4032,7 @@ org.opentripplanner.routing.algorithm.mapping.TransitSnapshotTest.test_trip_plan
     },
     {
       "arrivedAtDestinationWithRentedVehicle": false,
-      "durationSeconds": 1941,
```

Comment:
Why has this changed?

Reply:
This is a bit strange, but I went through th code to see that nothing is changed witch could influence the result of the router. So I guess the only plausible option is that two very similar itineraries are returned, and due to the filter only one was returned before.